CMakeLists.txt

```diff
@@ -520,14 +521,14 @@ if(ENABLE_PYTHON AND PYTHONLIBS_FOUND)
   # disable warnings on automatically generated interface code
   set_target_properties(crpropa-swig PROPERTIES COMPILE_FLAGS "-w")
   set_target_properties(crpropa-swig PROPERTIES OUTPUT_NAME "_crpropa")
-  target_link_libraries(crpropa-swig crpropa ${PYTHON_LIBRARIES})
+  target_link_libraries(crpropa-swig crpropa ${Python_LIBRARIES} ${Python_LIBRARY})
   add_dependencies(crpropa-swig crpropa-swig-wrapper)
 
-  install(DIRECTORY "${CMAKE_SOURCE_DIR}/python/crpropa" DESTINATION "${PYTHON_SITE_PACKAGES}")
-  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/crpropa.py" DESTINATION "${PYTHON_SITE_PACKAGES}/crpropa")
-  install(TARGETS crpropa-swig LIBRARY DESTINATION "${PYTHON_SITE_PACKAGES}/crpropa")
+  install(DIRECTORY "${CMAKE_SOURCE_DIR}/python/crpropa" DESTINATION ${CMAKE_INSTALL_PREFIX})
```

Comment:
This is a drastical change of installation behaviour. 
In this case the python package is not stored in the same directory as the other packages of an virtual environment. This should be changed to the old behaviour.

Reply:
I will have to check this again then. The reason for the change is that some tests were failing.

Reply:
At a second glance, after running some tests on my machine, the previous version was also problematic. We should not force installation at `Python_SITE_PACKAGES`. 
That is the reason why it was failing and why my `site-packages` folder was getting polluted against my will.
Do you have a more concrete suggestion to work around this, @JulienDoerner ?